src/scheduler/worker.rs

```diff
@@ -7,6 +7,8 @@ use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::mpsc::Sender;
 use std::sync::{Arc, Weak};
 
+const LOCALLY_CACHED_WORKS: usize = 1;
```

Comment:
This is changed from 10 to 1. Is this intentional?

Reply:
Yes. My benchmark results show that setting `LOCALLY_CACHED_WORKS` > 1 won't make performance better.

Reply:
But 1 is better than 0 (no local cache)? That's interesting...

Reply:
This local cache is specifically to address the "tail recursion" problem. i.e. A work packet `W` only creates another `W`,  which results a long `W->W->W->W->...` chain. So caching one packet locally is enough to make a worker consume all items in this chain, instead of pushing each item to the global pool.